EIPS/eip-7709.md

```diff
@@ -23,8 +23,8 @@ The `BLOCKHASH (0x40)` opcode currently assumes that the client has knowledge of
 
 | Parameter                 | Value  |
 | ------------------------- | ------ |
-| `FORK_TIMESTAMP`          | TBD    |
-| `HISTORY_STORAGE_ADDRESS` | TBD    |
+| `FORK_TIMESTAMP`          | `1717171717` |
+| `HISTORY_STORAGE_ADDRESS` | `0x0000F90827F1C53a10cb7A02335B175320002935` |
```

Comment:
[Link for other reviewers](https://eips.ethereum.org/EIPS/eip-2935#specification).